cumulusci/tasks/salesforce/tests/test_Deploy.py

```diff
@@ -43,10 +44,356 @@ def test_get_api__managed(self):
+                    expected.append(
+                        os.path.join(
+                            "./lwc/myComponent", lwc_component_file.get("name")
+                        )
+                    )
+
+            # add lwc component files not included in zip
+            for lwc_ignored_file in lwc_ignored_files:
+                with open(os.path.join(lwc_component_path, lwc_ignored_file), "w"):
+                    pass
+
+            # add lwc component sub-directory and files not included in zip
+            lwc_component_test_path = os.path.join(lwc_component_path, "__tests__")
+            os.mkdir(lwc_component_test_path)
+            with open(os.path.join(lwc_component_test_path, "test.js"), "w"):
+                pass
+
+            # add classes
+            classes_path = os.path.join(path, "classes")
+            os.mkdir(classes_path)
+            class_files = [
+                {
+                    "name": "MyClass.cls-meta.xml",
+                    "body": """<?xml version="1.0" encoding="UTF-8"?>
+<ApexClass xmlns="http://soap.sforce.com/2006/04/metadata">
+    <apiVersion>45.0</apiVersion>
+    <status>Active</status>
+</ApexClass>
+""",
+                },
+                {"name": "MyClass.cls"},
+            ]
+            for class_file in class_files:
+                with open(os.path.join(classes_path, class_file.get("name")), "w") as f:
+                    if class_file.get("body") is not None:
+                        f.write(class_file.get("body"))
+                    expected.append(os.path.join("./classes", class_file.get("name")))
+
+            # add objects
+            objects_path = os.path.join(path, "objects")
+            os.mkdir(objects_path)
+            object_file_names = ["Account.object", "Contact.object", "CustomObject__c"]
+            object_file_names.sort()
+            for object_file_name in object_file_names:
+                with open(os.path.join(objects_path, object_file_name), "w"):
+                    expected.append(os.path.join("./objects", object_file_name))
+
+            # add sub-directory of objects (that doesn't really exist)
+            objects_sub_path = os.path.join(objects_path, "does-not-exist-in-schema")
+            os.mkdir(objects_sub_path)
+            with open(os.path.join(objects_sub_path, "some.file"), "w"):
+                expected.append(
+                    os.path.join("./objects/does-not-exist-in-schema", "some.file")
+                )
+
+            # test
+            task = create_task(
+                Deploy,
+                {
+                    "path": path,
+                    "namespace_tokenize": "ns",
+                    "namespace_inject": "ns",
+                    "namespace_strip": "ns",
+                },
+            )
+
+            expected_set = set(expected)
+            actual_set = set(task._get_files_to_package(path))
+            self.assertEquals(expected_set, actual_set)
```

Comment:
You can use `self.assertSetEqual` and avoid needing to explicitly cast to `set` yourself.

Reply:
`assertSetEqual` [requires](https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertSetEqual) both inputs to be sets.

Changed to `assertSetEqual`